esp/src/eclwatch/HPCCPlatformWidget.js

```diff
@@ -415,7 +416,18 @@ define([
         _onAboutClose: function (evt) {
             this.aboutDialog.hide();
         },
-
+		
+        _onLogout: function (evt) {
+            xhr("esp/logout", {
```

Comment:
I don't believe that it needs to be but I will add and test it with /esp if you believe that is a better way to format it

As far as using an alternate URL you are able to reach ECLWatch, HOWEVER this does not affect my pull request. Also in doing so, the User has no access to any files. This should be fixed so the user can't get any of the alt URL's. I will open JIRA for this.